ui/raidboss/data/05-shb/raid/e1n.js

```diff
@@ -175,16 +175,15 @@
         'Primeval Stasis': 'Stase primordiale',
         'Pure Beam': 'Rayon purificateur',
         'Pure Light': 'Lumière purificatrice',
-        'Spear of Paradise': 'Lance du paradis',
+        'Spear Of Paradise': 'Lance du paradis',
         'Unto Dust': 'Déflagration',
-        'Vice and Virtue': 'Vice et vertue',
-        'Vice of Apathy': 'Péché d\'apathie',
-        'Vice of Vanity': 'Péché de vanité',
+        'Vice And Virtue': 'Vice et vertue',
+        'Vice Of Apathy': 'Péché d\'apathie',
```

Comment:
https://github.com/quisquous/cactbot/pull/1547#discussion_r452612032

Reply:
I'm not sure I understand, or made myself understood, so here is what it looks like if the timeline spelling is not the same. Sorry for the difficult understanding between us at times.
```
### Phase 3: more of the same
300.0 "Stase primordiale" sync /:▒den:3E3A:/ window 300,0
343.6 "Souffle de l'▒ternel" sync /:Primo-▒den:3DA5:/

354.0 "--targetable--" #MISSINGTEXT
360.0 "Vice And Virtue" sync /:Primo-▒den:44E4:/
360.8 "Vice Of Vanity" sync /:Primo-▒den:44E7:/
370.6 "Spear Of Paradise" sync /:Primo-▒den:3DA1:/
372.1 "Ravageur de paradis" sync /:Primo-▒den:3DA2:/
377.0 "--corner--" sync /:Primo-▒den:4683:/ #MISSINGTEXT
390.3 "Lumi▒re purificatrice" sync /:Primo-▒den:3DA3:/
396.2 "--center--" sync /:Primo-▒den:4683:/ #MISSINGTEXT
405.5 "Attaque Delta" sync /:Primo-▒den:44EA:/
406.1 "M▒ga Foudre ▒d▒nique" sync /:Primo-▒den:44ED:/
406.1 "M▒ga Glace ▒d▒nique" sync /:Primo-▒den:44EC:/
406.1 "M▒ga Feu ▒d▒nique" sync /:Primo-▒den:44EB:/
412.1 "--sync--" sync /:Primo-▒den:4683:/
418.7 "Translation dimensionnelle" sync /:Primo-▒den:3D9C:/
427.1 "Rayon purificateur x6" sync /:Primo-▒den:3D9D:/ duration 17.3
432.6 "Paradis perdu x3" sync /:Primo-▒den:3D9F:/ duration 11.2
435.1 "--corner--" sync /:Primo-▒den:4683:/ #MISSINGTEXT
448.4 "Lumi▒re purificatrice" sync /:Primo-▒den:3DA3:/
451.2 "--center--" sync /:Primo-▒den:4683:/ #MISSINGTEXT
457.8 "Translation dimensionnelle" sync /:Primo-▒den:3D9C:/
470.3 "Vice And Virtue" sync /:Primo-▒den:44E5:/
470.4 "Vice Of Apathy" sync /:Primo-▒den:44E8:/
476.9 "Brasier ▒d▒nique" sync /:Primo-▒den:3D97:/
488.4 "Gravit▒ ▒d▒nique" sync /:Primo-▒den:3D94:/

502.0 "Vice And Virtue" sync /:Primo-▒den:44E4:/ window 100,100 jump 360.0
502.7 "Vice Of Vanity"
512.5 "Spear Of Paradise"
514.0 "Ravageur de paradis"
519.0 "--corner--" #MISSINGTEXT
532.3 "Lumi▒re purificatrice"
538.2 "--center--" #MISSINGTEXT
547.4 "Attaque Delta"
547.9 "M▒ga Foudre ▒d▒nique"
547.9 "M▒ga Glace ▒d▒nique"
547.9 "M▒ga Feu ▒d▒nique"
554.1 "--sync--"
560.7 "Translation dimensionnelle"
569.2 "Rayon purificateur"
574.6 "Paradis perdu"
577.1 "--corner--" #MISSINGTEXT
590.4 "Lumi▒re purificatrice"
593.2 "--center--" #MISSINGTEXT
599.7 "Translation dimensionnelle"
612.3 "Vice And Virtue"
612.3 "Vice Of Apathy"

```

Reply:
This is just check_translation.py being incorrect.  You will notice that there is no `#MISSINGTEXT` comment on those lines.

I fixed this in https://github.com/quisquous/cactbot/commit/4ec92a699be578aecae8738adcf1abe159daaafb, thanks for letting me know.

Reply:
Do you want me to cancel this request?

Reply:
Can all of these missingTranslation still be removed without the test failing?

Reply:
I don't know if I understood correctly, but 2 points are certain:
Your fix work well, either your merged with the uppercases changes or not. it will work in both cases.
The deleted missingTranslation lines were no longer useful for multiple files 
```
$ python util/find_missing_translations.py -f e4n.js -l ja
C:\Users\Bauer\Documents\GitHub\cactbot\ui\raidboss\data\05-shb\raid\e4n.js: missingTranslations set true when not needed
```
Tell me what you want me to do.

Reply:
If you want to remove unnecessary missingTranslations lines either in this pull request or other pull requests, I think that would be good.  I don't think capitalization should be changed.